hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/Dialogs/ReviewOrgUnitChangesInfos.tsx

```diff
@@ -53,56 +70,72 @@ export const ReviewOrgUnitChangesInfos: FunctionComponent<Props> = ({
     const styles = useStatusStyles(changeRequest?.status);
     if (isFetchingChangeRequest || !changeRequest) return null;
     const { status } = changeRequest;
+    const labelStyle = { ...styles.label, ...styles.cell };
+    const valueStyle = { ...styles.value, ...styles.cell };
```

Comment:
It works as is, but it's not consistent to memoize `styles` (through the hook) and not `labelStyle` and `valueStyle` which are also objects.

You can probably tweak the hook to directly return the classes you need by not returning the object immediately